doc/content/getting-started/installation/configuration/ttn-lw-stack-docker-enterprise.yml

```
@@ -35,6 +35,10 @@ is:
       is:
         base-url: 'https://thethings.example.com/api/v3'
 
+  # If running a multi-tenant environment, configure tenant admin keys
+  tenancy:
+    admin-keys:                   # generate 32 bytes (openssl rand -hex 32)
```

For consistency with the rest of the config

```suggestion
    admin-keys: ''                  # generate 32 bytes (openssl rand -hex 32)
```